python/ray/autoscaler/_private/updater.py

```diff
@@ -303,6 +305,11 @@ def do_update(self):
         node_tags = self.provider.node_tags(self.node_id)
         logger.debug("Node tags: {}".format(str(node_tags)))
 
+        if self.provider.provider_config:
```

Comment:
I think we'd also want to check if we're using AWSNodeProvider here (wouldn't want the initialization of the `CloudwatchHelper` to crash, say, a GCP or Azure Ray cluster.)

Reply:
Thanks Dmitri, added provider type check.